src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/caseupdate/services/UpdatedCaseValidator.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.caseupdate.services;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.common.model.out.SampleCase;
+
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+
+@Component
+public class UpdatedCaseValidator {
```

Comment:
I think it's the request that should be validated, instead of the result object that this service creates.

Reply:
The request will be validated in the next PR using `Assert.notEmpty`.

I deliberately extracted response validation as it is returning `emptyList()` atm and can be changed later

Reply:
What are we going to validate about the response?

Reply:
At the moment - nothing. This change will add certain level of structure to the code and make it possible to add real warnings when we need

Reply:
What will we validate about the response in the future?